tests/driver_soft_spi/Makefile

```diff
@@ -0,0 +1,9 @@
+APPLICATION = driver_soft_spi
+BOARD ?= stm32f4discovery
+include ../Makefile.tests_common
+
+USEMODULE += soft_spi
+
+CFLAGS += -I$(CURDIR)
```

Comment:
The CFLAGS or the USEMODULE?

The CFLAGS ist necessary because else the compiler would disregard the soft_spi_params.h in the project directory.